content/guides/monitoring.md

```diff
@@ -336,7 +336,50 @@ right people get notified so the problem can be resolved as soon as possible.
    anytime the monitor renotifies. The original message will be included as
    well.
 
-## Monitor FAQs 
+
+### Message template variables
+
+Message template variables can be used to customize your monitor notifications.
+This feature is supported in all monitor types. There are two primary use cases
+for template variables: 1) displaying a different message depending on the
+notification type (e.g. triggered, recovered, no data) and 2) incorporating the
+triggering scope into the message of Multi Alerts.
+
+1. **Conditional variables for different notification types**: You can have a
+    monitor event display a different message depending on whether the event is a
+    trigger, resolve, or no data notification. These variables use simple if-else
+    logic with the following syntax:
+
+        {{#var}} This text will show {{/var}} only if the variable `var` is true.
+        {{^var}} This text will show {{/var}} only if `var` is NOT true.
+
+    The conditional variables available are is_alert, is_recovery, and is_no_data.
+    These can also be seen in the "Use message template variables" help box in
+    Step 3 of the monitor editor.
```

Comment:
I think it would be beneficial to add a real example here. Just something simple but that shows a real use of the conditional variables